src/impl/util.js

```diff
@@ -64,6 +64,10 @@ export class Util {
     return ('0'.repeat(n) + input).slice(-n);
   }
 
+  static padEnd(input, n = 9) {
+    return (input + '0'.repeat(n)).slice(0, n);
```

Comment:
This might be better as

```js
input.length > n ? input : input + '0'.repeat(n - input.length)
```

Reply:
Oh, I see. You were just using the same idiom as `pad` just above it. Nevermind, just leave it :)

Reply:
Perhaps `padEnd` is the wrong name for this. The intent of this is similar to `pad` above in that if there were more than `n` characters, I want only the first `n` characters. `pad('123')` above will return `'23'` and `padEnd('123', 2)` should return `'12'`.

Reply:
As long as we're consistent. I renamed `pad` to `padStart`. It's possible they should both have clearer names but it seems OK for now.